tests/integration/renderer/test_persistence.py

```diff
@@ -526,3 +532,189 @@ def set_out(val):
         dash_duo.wait_for_text_to_equal(".out", "")
 
         dash_duo.find_element("#btn").click()
+
+
```

Comment:
Test persistence for DatePickerSingle and DatePicker range in callbacks, in conjunction with https://github.com/plotly/dash-core-components/pull/848.

Reply:
While I think it's good to have at least one test that checks that persistence works as expected with and without a `part`, the tests validating the behavior of DPSingle and DPRange should be moved to DCC.

Reply:
We could create a new test component similar to https://github.com/plotly/dash/tree/dev/%40plotly/dash-generator-test-component-standard that has both (1) prop part, (2) no prop part for two different persisted props, and we can validate the behavior against that component instead.

Reply:
b063d60 to e4026e6